examples/nginx-https/README.md

```diff
@@ -0,0 +1,99 @@
+<!-- BEGIN MUNGE: UNVERSIONED_WARNING -->
+
+<!-- BEGIN STRIP_FOR_RELEASE -->
+
+<h1>*** PLEASE NOTE: This document applies to the HEAD of the source
+tree only. If you are using a released version of Kubernetes, you almost
+certainly want the docs that go with that version.</h1>
+
+<strong>Documentation for specific releases can be found at
+[releases.k8s.io](http://releases.k8s.io).</strong>
+
+<!-- END STRIP_FOR_RELEASE -->
+
+<!-- END MUNGE: UNVERSIONED_WARNING -->
+# Nginx https service
```

Comment:
we could merge the 2, simplenginx uses the standard nginx container which listens on port 80 and doesn't understand https. The real point of this example is to include it in a user guide where we describe multi-port (http/https) services. This README is more about having a working https server and less about running nginx in a container.